src/test/java/com/fasterxml/jackson/core/read/LocationOffsetsTest.java

```diff
@@ -3,6 +3,11 @@
 import com.fasterxml.jackson.core.*;
 import com.fasterxml.jackson.core.json.JsonFactory;
 
+import java.io.IOException;
+import java.util.Base64;
```

Comment:
Can not use until 3.0 (part of JDK 8). Same for a few other things, will make necessary changes.

Reply:
I just replaced it with a `generateRandomAlpha` method.